### PR TITLE
Adfs Module: Improve Inputs and Outputs for the AdfsClaimDescription Cmdlets

### PR DESCRIPTION
**Pull request by X-Guardian**
This PR improves the descriptions for the `Inputs` and `Outputs` section of the following `AdfsCertificate` cmdlets:

- Add-AdfsClaimDescription
- Get-AdfsClaimDescription
- Remove-AdfsClaimDescription
- Set-AdfsClaimDescription

It also removes several explicit references to AD FS 2.0.

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/adfs/Get-AdfsClaimDescription.md [hunk 1]
```diff
@@ -109,15 +109,18 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 ## INPUTS
 
-### None
+### System.String
+
+String objects are received by the *ClaimType*, *Name* and *ShortName* parameters.
```
Comment (JohanFreelancer9) [inline docset/windows/adfs/Get-AdfsClaimDescription.md]:
```suggestion
String objects are received by the *ClaimType*, *Name*, and *ShortName* parameters.
```

File: docset/windows/adfs/Remove-AdfsClaimDescription.md [hunk 2]
```diff
@@ -176,14 +176,23 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 ## INPUTS
 
 ### Microsoft.IdentityServer.PowerShell.Resources.ClaimDescription
-You can pipe a class structure that represents a claim description object for the Federation Service.
+
+A ClaimDescription object is received by the *TargetClaimDescription* parameter.
+
+### System.String
+
+String objects are received by the *TargetClaimType*, *TargetName* and *TargetShortName* parameters.
```
Comment (JohanFreelancer9) [inline docset/windows/adfs/Remove-AdfsClaimDescription.md]:
```suggestion
String objects are received by the *TargetClaimType*, *TargetName*, and *TargetShortName* parameters.
```

File: docset/windows/adfs/Set-AdfsClaimDescription.md [hunk 3]
```diff
@@ -289,11 +289,18 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 ## INPUTS
 
 ### Microsoft.IdentityServer.PowerShell.Resources.ClaimDescription
-A class structure for representing a claim description object for the Federation Service.
+
+A ClaimDescription object is received by the *TargetClaimDescription* parameter.
+
+### System.String
+
+String objects are received by the *TargetClaimType*, *TargetName* and *TargetShortName* parameters.
```
Comment (JohanFreelancer9) [inline docset/windows/adfs/Set-AdfsClaimDescription.md]:
```suggestion
String objects are received by the *TargetClaimType*, *TargetName*, and *TargetShortName* parameters.
```

**Pull request merged**